users/serializers.py

```diff
@@ -15,6 +15,22 @@ class Meta:
         model = CustomUser
         fields = '__all__'
 
+    def validate_email(self, email):
+        email_old = email
+        email = allauth_get_adapter().clean_email(email)
+        domain = email.split('@')[1]
+        domain_list = ["codepoets.it",]
```

Comment:
Same as few comments above. If it would be in constants then you would just import it from there

Reply:
Done